homeassistant/components/overkiz/binary_sensor.py

```diff
@@ -105,6 +105,22 @@ class OverkizBinarySensorDescription(BinarySensorEntityDescription):
         )
         == 1,
     ),
+    OverkizBinarySensorDescription(
+        key=OverkizState.CORE_HEATING_STATUS,
+        name="Heating Status",
+        device_class=BinarySensorDeviceClass.HEAT,
+        value_fn=lambda state: state == OverkizCommandParam.ON,
+    ),
+    OverkizBinarySensorDescription(
+        key=OverkizState.MODBUSLINK_DHW_ABSENCE_MODE,
+        name="Absence Mode State",
```

Comment:
I am not sure if we should add 'State' here, or just "Absence Mode".

Reply:
Done